packages/test-utils/src/legacy-cli/main.ts

```diff
@@ -184,10 +194,18 @@ async function setMetadata(
       await retryWithExponentialBackoff(
         () => client.setRecordingMetadata(recordingId, metadata),
         e => {
-          debug("Failed to set recording metadata. Will be retried:  %j", e);
+          logger.error("SetMetadata:WillRetry", {
+            recordingId,
+            errorMessage: getErrorMessage(e),
+          });
         }
       );
     } catch (e) {
+      logger.error("SetMetadata:Failed", {
+        recordingId,
+        errorMessage: getErrorMessage(e),
```

Comment:
I don’t think it’s a good idea to strip down to just the error name when logging into Grafana. The stack trace is helpful for debugging the problem.